spec/haskell/src/SEL4/Machine/Hardware/ARM/Exynos4210.hs

```diff
@@ -164,7 +164,7 @@ cacheInvalidateL2RangeCallback _ _ _ = return ()
 cacheCleanL2RangeCallback :: Ptr CallbackData -> PAddr -> PAddr -> IO ()
 cacheCleanL2RangeCallback _ _ _ = return ()
 
--- For the ARM1136
+-- Overwritten in Isabelle
```

Comment:
I don't like this phrase. Don't we have an existing one that makes sense already? I'm not sure it's "overwritten in Isabelle", but rather it's clobbered by the abstract spec, or (non-exclusive) it's not imported by the translator.

Reply:
Or maybe unused by the design/exec spec ...

Reply:
Ah, this is what the `error` comment was with regard to. Yes, might be good to error them out.